cargo-concordium/src/build.rs

```diff
@@ -451,6 +451,37 @@ fn write_schema_json(
     Ok(())
 }
 
+/// Write the template of the schema to a file or print it
+/// to the console if `out` is None.
+pub fn write_schema_template(
+    out: Option<PathBuf>,
+    schema: &VersionedModuleSchema,
+) -> anyhow::Result<()> {
+    match out {
+        // writing the template of the schema to a file
+        Some(out) => {
+            println!(
+                "   Writing the template of the schema to {}.",
+                out.display()
+            );
+            if let Some(out_dir) = out.parent() {
+                fs::create_dir_all(out_dir).context(
```

Comment:
Does this work correctly for relative paths where `.parent` returns `Some("")`?

Reply:
yes, it works for relative paths:
```
cargo concordium build --schema-template-out ./myTemplate.txt
```

Reply:
And I assume also
```
cargo concordium build --schema-template-out myTemplate.txt
```

Reply:
yes, that also works.